zsh/README.md

```diff
@@ -21,18 +28,40 @@
 ### Other OS
 
 `https://github.com/robbyrussell/oh-my-zsh/wiki/Installing-ZSH`
+</details>
 
 ## Install oh-my-zsh
 
 `sh -c "$(curl -fsSL https://raw.githubusercontent.com/robbyrussell/oh-my-zsh/master/tools/install.sh)"`
 
-## Addons & plugins flow
+## Install Preferd Fonts - `nerd-fonts` - fix it
+<details> <summary>Follow the steps here:</summary>
+
+- `git clone https://github.com/ryanoasis/nerd-fonts.git --depth 1`
+- Go to repo. folder
+- run `./install.sh`
+
+**OR**
+
+- `cd ~/Library/Fonts && curl -fLo "Meslo LG L Regular Nerd Font Complete.ttf" https://github.com/ryanoasis/nerd-fonts/blob/master/patched-fonts/Meslo/L/Regular/complete/Meslo%20LG%20L%20Regular%20Nerd%20Font%20Complete.ttf`
+
+
+- Set 1 of the above `nerd-fonts` in the terminal settings:
+- in Iterm go to preferences -> prfofiles tab-> text tab -> font
+</details>
+
+## Install Addons & plugins flow
+<details> <summary>Follow the steps here:</summary>
 
 - `git clone https://github.com/zsh-users/zsh-autosuggestions ${ZSH_CUSTOM:-~/.oh-my-zsh/custom}/plugins/zsh-autosuggestions`
+- `git clone https://github.com/zsh-users/zsh-syntax-highlighting.git ${ZSH_CUSTOM:-~/.oh-my-zsh/custom}/plugins/zsh-syntax-highlighting`
+- `source ${ZSH_CUSTOM:-~/.oh-my-zsh/custom}/plugins/zsh-syntax-highlighting`
```

Comment:
The `source` command should go inside the `.zshrc` and before the `plugins` section